src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1001,4 +1001,72 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return OperationKind.ExpressionStatement
         End Function
     End Class
+
+    Partial Class BoundAddRemoveHandlerStatement
+        Implements IEventAssignmentExpression
```

Comment:
I'm fixing this as @JohnHamby suggested, but in #7352 instead of a separate PR, to save myself some future merging trouble.